mono/mini/local-propagation.c

```diff
@@ -672,6 +672,9 @@ mono_local_cprop (MonoCompile *cfg)
 						if ((def->opcode == OP_I8CONST) && (sizeof (gpointer) == 4)) {
 							ins->inst_ls_word = def->inst_ls_word;
 							ins->inst_ms_word = def->inst_ms_word;
+						} else if (regtype == 'l' && sizeof (gpointer) == 4) {
```

Comment:
Shouldn't this use the new TARGET_ defines you added?

Reply:
This is for 2018-06.